planex/cmd/pin.py

```diff
@@ -158,11 +162,21 @@ def main(argv=None):
     args = parse_args_or_exit(argv)
 
     if args.base is not None and args.base_commitish is None:
-        print("Error: --base_commitish is required if --base is used.")
+        print("Error: --base_commitish is required if --base is used.",
+              file=sys.stderr)
         sys.exit(1)
 
     package_name = args.package
     xs_path = os.getcwd()
     pin = make_pin(args, xs_path, package_name)
 
     print(json.dumps(pin, indent=2))
+
+    if args.to:
+        path = os.path.basename(args.to)
+        if os.path.exists(path):
+            with open(args.to, "w") as out:
+                json.dump(pin, out, indent=2)
+        else:
+            print("Error: path {} does not exist.".format(path),
```

Comment:
It might be better to use `sys.exit` here.   You can pass it an exit message instead of just an exit code.   If you still need to use print, please at least exit with a non-zero error code afterwards.